flask_sqlalchemy/__init__.py

```diff
@@ -36,7 +36,10 @@
 
 # the best timer function for the platform
 if sys.platform == 'win32':
-    _timer = time.clock
+    if sys.version_info()[:2] > (3, 3):
```

Comment:
See: https://docs.python.org/3/library/time.html#time.perf_counter

```suggestion
    if sys.version_info()[:2] >= (3, 3):
```

Reply:
Oh, thanks for getting into this.